test/VampIR/Compilation/Base.hs

```diff
@@ -13,3 +13,19 @@ vampirCompileAssertion paramsNum mainFile dataFile step = do
   tab <- (^. coreResultTable) . snd <$> runIO' entryPoint upToCore
   coreVampIRAssertion' tab toVampIRTransformations mainFile dataFile step
   vampirAssertion' paramsNum tab dataFile step
+
+vampirCompileErrorAssertion ::
+  Path Abs File ->
+  (String -> IO ()) ->
+  Assertion
+vampirCompileErrorAssertion mainFile step = do
+  step "Translate to JuvixCore"
+  entryPoint <- defaultEntryPointCwdIO mainFile
+  r <- runIOEither entryPoint upToCore
+  case r of
+    Left _ -> assertBool "" True
```

Comment:
Suggestion: `Assertion` is just an alias for `IO ()`, so you can use `return ()` here instead of `assertBool "" True`